day_7/ceasar_cipher_attempt.rb

```diff
@@ -0,0 +1,41 @@
+class CeasarCipher
+
+  def initialize
+  end
+
```

Comment:
I like an OOP approach to this problem! Not having any attributes makes this a bit less OOPy.... its not wrong, but I think you could experiment with passing info into your instance creation to give your class a few attributes to be held there. But, ultimately, up to you!